examples/neg/exists_extrudes.effekt

```diff
@@ -0,0 +1,15 @@
+type Exists {
+  Op[Z](el: Z, combine: (Z, Z) => Z at {})
+}
+
+def test2extruding() = {
+  val intBox = Op(1, box { (x, y) => x + y });
+
+  val x = intBox match {
+    case Op(el, combine) => el
+  }
+  x + 2 // ERROR Cannot typecheck
```

Comment:
We do not check for extrusion, but since the existentials are fresh it shouldn't be possible to do anything with them. Inferred types will just mention strange, not bound, type parameters.